tests/test_api.py

```diff
@@ -69,7 +69,7 @@ def test_scene_search(self):
         result = self.api.scene_search("landsat_tm_c2_l1", max_results=1, metadata_type=None)
 
         assert result["recordsReturned"] == 1
-        assert result["totalHits"] == 2940421
+        assert result["totalHits"] > 2940000
```

Comment:
This number is likely to increase with time. So another option is to set an end date for the request so that it ignores new results after a given date.